doc/command_ref.rst

```diff
@@ -207,7 +210,10 @@ Options
     Enable the listed plugins specified by names or globs.
 
 ``--enablerepo=<repoid>``
-    Enable additional repositories by an id or a glob.
+    Temporarily enable additional repositories for the purpose of the current dnf command.
+    Accepts an id, a comma-separated list of ids, or a glob of ids. This option can be
+    specified multiple times.
+    To permanently enable new repositories, use `dnf config-manager --enable <repo id>`
```

Comment:
Unfortunately, `dnf config-manager` is actually not yet part of the dnf core, so we can't refer to it here, as it's entirely possible that isn't present at all.